hermes-console-vue/src/views/subscription/manage-messages-card/ManageMessagesCard.spec.ts

```diff
@@ -1,12 +1,56 @@
+import { appConfigStoreState } from '@/dummy/store';
+import { createTestingPinia } from '@pinia/testing';
+import { dummySubscription } from '@/dummy/subscription';
+import { dummyTopic } from '@/dummy/topic';
 import { render } from '@/utils/test-utils';
 import ManageMessagesCard from '@/views/subscription/manage-messages-card/ManageMessagesCard.vue';
+import userEvent from '@testing-library/user-event';
 
 describe('ManageMessagesCard', () => {
-  it('should render manage messages card', () => {
+  it('should open confirmation popup when retransmit button is clicked', async () => {
+    // given
+    const user = userEvent.setup();
+
+    // when
+    const { getByTestId, getByText } = render(ManageMessagesCard, {
+      props: {
+        topic: dummyTopic.name,
+        subscription: dummySubscription.name,
+      },
+      testPinia: createTestingPinia({
+        initialState: {
+          appConfig: appConfigStoreState,
```

Comment:
if you are not overriding store state, you can just use `createTestingPiniaWithState()`